typescript/nextjs/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@bufbuild/protobuf": "1.8.0",
+    "@connectrpc/connect": "1.3.0",
+    "@connectrpc/connect-web": "1.3.0",
+    "@substreams/core": "0.16.0",
+    "@substreams/manifest": "^0.0.9",
+    "@uidotdev/usehooks": "^2.4.1",
+    "next": "14.1.4",
+    "react": "^18",
+    "react-dom": "^18"
+  },
+  "devDependencies": {
+    "@types/node": "^20",
```

Comment:
That seems superflous in this context, no reason to bring `@types/node` in a Browser project.